praw/models/reddit/redditor.py

```diff
@@ -119,9 +129,17 @@ def __init__(self, reddit, name=None, _data=None):
         self._listing_use_sort = True
         if name:
             self.name = name
-        self._path = API_PATH["user"].format(user=self)
+        elif fullname:
+            self._fullname = fullname
+
+    def _fetch_username(self, fullname):
+        return self._reddit.request(
+            "GET", API_PATH["user_by_fullname"], {"ids": fullname}
+        )[fullname]["name"]
```

Comment:
Unless there is a specific reason for doing it this way, could you please use the `.get()` method on `self._reddit`? That would look like

    return self._reddit.get(API_PATH['user_by_fullname'], params={'ids': fullname})[fullname]['name']

I do see that `self._reddit.request` is currently used in two other places in the file, however I consider using `self._reddit.get` to be cleaner form.

Reply:
Ahh yep. I didn't even realise there was a `.get` since I just looked around for similar calls! Fixed